galaxy/api/access.py

```diff
@@ -391,6 +391,22 @@ def can_delete(self, obj):
         return False
 
 
+class SessionEventAccess(BaseAccess):
+    model = models.SessionEvent
+
+    def can_read(self, obj):
+        return True
```

Comment:
Do we really want to allow anybody access to our internal session tracking information?
I would say it should be push-only.

Reply:
if we change `SessionEvent` to store session hashes, does it matter if people can read the hashes?